tools/clang/test/CodeGenSPIRV_Lit/intrinsics.dot4add.hlsl

```diff
@@ -1,23 +1,28 @@
-// RUN: not %dxc -E frag -T ps_6_4 -fspv-target-env=vulkan1.1 -fcgl  %s -spirv  2>&1 | FileCheck %s
+// RUN: %dxc -E main -T ps_6_4 -fspv-target-env=vulkan1.1 -fcgl  %s -spirv  2>&1 | FileCheck %s
 
-uint frag(float4 vertex
-          : SV_POSITION) : SV_Target {
-  uint acc = 0;
-
-  // CHECK: 8:14: error: dot4add_u8packed intrinsic function unimplemented
-  acc += 1 + dot4add_u8packed(vertex.x, vertex.y, uint(vertex.y));
+float2 f2;
```

Comment:
Since this var isn't really involved in the test behavior, can you remove it for simplicity?

Reply:
Removed.